casr/src/bin/casr-core.rs

```diff
@@ -199,22 +189,18 @@ fn main() -> Result<()> {
 
     let executable_path = PathBuf::from(
         matches
-            .value_of("executable")
+            .get_one::<String>("executable")
             .unwrap()
             .chars()
             .map(|c| if c == '!' { '/' } else { c })
             .collect::<String>(),
     );
-    let pid = matches.value_of("pid").unwrap().parse::<i32>().unwrap();
-    let culimit = matches
-        .value_of("core")
-        .unwrap_or("0")
-        .parse::<i32>()
-        .unwrap_or(-1);
-    let uid = matches.value_of("uid").unwrap().parse::<u32>().unwrap();
-    let gid = matches.value_of("gid").unwrap().parse::<u32>().unwrap();
+    let pid = *matches.get_one::<i32>("pid").unwrap();
+    let culimit = *matches.get_one::<i32>("core").unwrap_or(&-1);
```

Comment:
there was different logic: when core is not present, the default value is 0, we may just set default value with clap